shared/django_apps/core/migrations/0050_repository_tests_enabled.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.13 on 2024-05-27 11:39
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    """
+    BEGIN;
+    --
+    -- Add field test_analytics_enabled to repository
+    --
+    ALTER TABLE "repos" ADD COLUMN "test_analytics_enabled" boolean DEFAULT false NULL;
+    ALTER TABLE "repos" ALTER COLUMN "test_analytics_enabled" DROP DEFAULT;
+    COMMIT;
+    """
+
+    dependencies = [
+        ("core", "0049_increment_version"),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
let's use the RiskyAddField for this since its updating the repos table

Reply:
interesting, anything we add to repo is a risky add?

Reply:
Generally yes (similar with other widely used tables like owners and commits) because the migration will timeout because it will be blocked for a long time by other operations touching the table, but if your new column is `(default=False, null=True)` then in theory it should be ok. But I'm not sure that's always the case and past migrations have gotten me paranoid. So IMO just be safe and do risky.

Reply:
Just in case, the "new" way we get these into prod is by merging this code and then updating the shared version in api/worker. When you do so, schedule sometime with the infra team to run the migration after hours, cause if you merge the code itself, the migrations won't be applied, so they need to run the SQL for you then.

Reply:
Thank you both!